.env.example

```
@@ -31,6 +31,8 @@ WEBHOOK_CIRCUIT_BREAKER_FAILURE_THRESHOLD=5
 WEBHOOK_CIRCUIT_BREAKER_SIGNATURE_THRESHOLD=3
 WEBHOOK_CIRCUIT_BREAKER_WINDOW_MINUTES=30
 WEBHOOK_CIRCUIT_BREAKER_COOLDOWN_MINUTES=30
+WEBHOOK_MERGE_GATE_REQUIRED_CHECKS=ci / test,lint
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The WEBHOOK_MERGE_GATE_REQUIRED_CHECKS key should go before the WEBHOOK_REPO_MUTATING_BUDGET_LIMIT key

(UnorderedKey)